jerry-core/parser/js/js-parser-expr.c

```diff
@@ -613,7 +683,24 @@ parser_parse_class_body (parser_context_t *context_p, /**< context */
       uint32_t accessor_status_flags = PARSER_FUNCTION_CLOSURE | PARSER_ALLOW_SUPER;
       accessor_status_flags |= (is_getter ? PARSER_IS_PROPERTY_GETTER : PARSER_IS_PROPERTY_SETTER);
 
-      lexer_expect_object_literal_id (context_p, LEXER_OBJ_IDENT_ONLY_IDENTIFIERS);
+      uint8_t ident_opts = LEXER_OBJ_IDENT_ONLY_IDENTIFIERS;
+
+      if (lexer_check_next_character (context_p, LIT_CHAR_HASHMARK))
+      {
+        lexer_next_token (context_p);
+        context_p->token.flags |= LEXER_NO_SKIP_SPACES;
```

Comment:
next token and adding no skip_space are frequently used together, i suggest to create a function for it, like `lexer_consume_hasmark.`